README.md

```diff
@@ -25,83 +92,104 @@ Edit your pom file and add the plugin:
+* Set the target to `ios` (for iOS devices) in the `pom.xml`:
 
-Run:
+```
+<artifactId>client-maven-plugin</artifactId>
+<configuration>
+    <target>ios</target>
+    <mainClass>${mainClassName}</mainClass>
+</configuration>
+```
 
-    mvn client:link
-    
-The results will be available at `target/client/$targetPlatform/$AppName`.
-    
-#### `client:build`
+* Build the native image:
 
-This goal simply combines `client:compile` and `client:link`.
-    
-#### `client:run`
+```
+mvn clean client:build
+```
 
-Runs the executable in the target platform.
+* Run the app on the connected iOS device:
 
-Run:
+```
+mvn client:run
+```
 
-    mvn client:run
-    
-Or run directly the application from command line:
+* Or package and create an IPA file to submit to TestFlight or to the App Store:
 
-    target/client/$targetPlatform/$AppName/$AppName    
-    
-It will create a distributable native application.
+```
+mvn client:package
+```
 
-#### `client:package`
+**Note**: Since all java bytecode is translated to native code, the compilation step can take a long time, and it requires a fair amount of memory.
 
-Create a package of the executable in the target platform.
+**Note**: In order to deploy apps to an iOS device, you need a valid iOS provisioning profile, as explained in the [documentation](https://docs.gluonhq.com/client/#_ios_deployment).
 
-Run:
+#### Android
 
-    mvn client:package
-    
-For instance, on iOS, this can be used to create an IPA.
+* Set the target to `android` (for android devices) in `pom.xml`:
+
+```
+<artifactId>client-maven-plugin</artifactId>
+<configuration>
+    <target>android</target>
+    <mainClass>${mainClassName}</mainClass>
+</configuration>
+```
 
-## Requirements
+* Build the native image:
 
-At this moment the plugin is in beta, and supports Linux, Mac OS X and iOS platforms for now.
+```
+mvn clean client:build
+```
 
-To use the plugin to develop and deploy native applications on Mac or iOS platforms, you need a Mac with MacOS X 10.13.2 or superior, and Xcode 9.2 or superior, available from the Mac App Store. Once Xcode is downloaded and installed, open it and accept the license terms.
+* Package and create an APK file:
 
-For now, only JDK 11 is supported. Any JDK 11 distribution that doesn't bundle JavaFX is valid, like:
+```
+mvn client:package
+```
 
-- [OpenJDK 11.0.2](https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_osx-x64_bin.tar.gz)
+* Install the APK file on a connected Android device:
 
-- [AdoptOpenJDK 11.0.3](https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.3%2B7/OpenJDK11U-jdk_x64_mac_hotspot_11.0.3_7.tar.gz)
+```
+mvn client:install
+```
 
-Once downloaded and installed, set `JAVA_HOME` pointing to that JDK 11.
+* Run the installed app on the connected Android device:
+
+```
+mvn client:run
+```
+
+**Note**: Since all java bytecode is translated to native code, the compilation step can take a long time, and it requires a fair amount of memory.
```

Comment:
This note can be moved a bit higher, after the `mvn clean client:build` command?

Reply:
Ok, moved